### DISCUSSION TITLE
FluxNode runs only on vintage OS. Why?

### DISCUSSION THREAD

**Discussion by crystal-way** [RunOnFlux/fluxnode-multitool#146] [General]
I was surprised..... After couple of months people will start migrations to the new 24.04LTS, while FLUX - project of the future web, not supported on 2 years old 22.04 OS. Why?

🚨 ERROR: OS version Ubuntu - jammy not supported
🚨 AVX CPU instruction set not found and is required to use MongoDB on jammy
🚨 Ubuntu 20.04 LTS is the recommended OS version... please re-image and retry installation
🚨 Installation stopped...

p.s. discord link on the official website also is not valid.

**Comment by XK4MiLX** [comment]
FluxOS is working fine on Ubuntu 22.04 if your hardware requirements are met. More info here
``https://www.mongodb.com/docs/manual/administration/production-notes/#std-label-prod-notes-supported-platforms``

**Comment by crystal-way** [comment]
Well i see this:

"Version: v7.7
Branch: master
OS: Ubuntu 16/18/19/20, Debian 9/10"

I don't see here Ubuntu 22, as well i see error message saying: "Ubuntu 20.04 LTS is the recommended OS version... please re-image and retry installation". :(

The link you gave me is for mongodb, not for flux node not for multitoolbox. 
 see that Flux is compatible with arm64. I'm trying to run on Orange pi with 8 core cpu, 16GB RAM and SSD. All benchmarks saying i'm good to go. Also i have installed and running fine docker, it's already working serving other purpose. 

I can't find any other installation on Flux page as well as no documentation for installation steps. The link for discord on official page also not working. Have no idea where i can get manual or more information. 

If you know something and could share with me, i would be thankful.

**Reply by XK4MiLX** [reply]
MongoDB is component of FluxOs .
MongoDB on arm64 requires the ARMv8.2-A or later microarchitecture. 
Orange pi 5 and Raspberry pi 5 met that requirements   min.  Ubuntu 23.10, Debian 12

**Reply by crystal-way** [reply]
Thank you for your reply.

So i've reinstalled orange pi, now i have this OS:

lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm


But for some reason two packages where not installed. Any idea how to fix it?

![flux_node](https://github.com/RunOnFlux/fluxnode-multitool/assets/35271080/d646b1cb-5931-4a8d-a61a-06da07cef4e0)


Thank you.

**Reply by XK4MiLX** [reply]
Use this one after install use option 11 and switch to development mode
```bash -i <(curl -s https://raw.githubusercontent.com/RunOnFlux/fluxnode-multitool/master/multitoolbox.sh) pi5```
